entrypoint.sh

```diff
@@ -17,14 +18,26 @@ if [[ $env == "sit" || $env == "uat" ]]; then
   jq --arg a "https://harmony.uat.earthdata.nasa.gov" '.URL.URLValue = $a' cmr/cmr.json > cmr/cmr.json.tmp && mv cmr/cmr.json.tmp cmr/cmr.json
 fi
 
-if [[ "${LAUNCHPAD_TOKEN_SIT}" ]] && [[ $env == "sit" ]]; then
-  umms_updater -d -f cmr/cmr.json -a cmr/${env}_associations.txt -p ${provider} -e ${env} -t $LAUNCHPAD_TOKEN_SIT -to $timeout
-elif [[ "${LAUNCHPAD_TOKEN_UAT}" ]] && [[ $env == "uat" ]]; then
-  umms_updater -d -f cmr/cmr.json -a cmr/${env}_associations.txt -p ${provider} -e ${env} -t $LAUNCHPAD_TOKEN_UAT -to $timeout
-elif [[ "${LAUNCHPAD_TOKEN_OPS}" ]] && [[ $env == "ops" ]]; then
-  umms_updater -d -f cmr/cmr.json -a cmr/${env}_associations.txt -p ${provider} -e ${env} -t $LAUNCHPAD_TOKEN_OPS -to $timeout
+if disable_removal == 'true'; then
```

Comment:
can't we just pass in disable_removal during the command argument instead of making an if else

Reply:
The -r input is a bool, that gets value if the argument is there or not, just like the -d for debug.